### PR TITLE
Upload

### PR DESCRIPTION
**Pull request by Dom-in**
## Statcord.com
Viewing Bot Statatistics
enter image description here
Viewing Statcord Docs
enter image description here

##  Quacky.xyz
Viewing Main Page
enter image description here
Viewing Vote Page
enter image description here

**IMAGRE IS THERE BUT DISCORD STILL CHECKING IT**

### CODE REVIEW

File: websites/Q/Quacky/presence.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const presence = new Presence({
+  clientId: "882955333121757236"
+}),
+
```
Comment (EncryptedDev) [inline websites/Q/Quacky/presence.ts:4]: Remove empty line

File: websites/Q/Quacky/presence.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  if (document.location.hostname === "docs.quacky.xyz") 
+    presenceData.details = "Viewing Docs";
+  
+
+  if (presenceData.details === null) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else 
+    presence.setActivity(presenceData);
+
```
Comment (EncryptedDev) [inline websites/Q/Quacky/presence.ts:40]: You're never setting the trayTitle(), so not needed 
```suggestion
presence.setActivity(presenceData);
```

File: websites/Q/Quacky/presence.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      presenceData.details = `Viewing Page:`;
+      presenceData.state = "Bots List";
+    } else if (document.location.pathname.includes("/add")) {
+      presenceData.details = `Viewing Page:`;
+      presenceData.state = `Add Bot`;
+    }
+  }
+
+  if (document.location.hostname === "docs.quacky.xyz") 
+    presenceData.details = "Viewing Docs";
```
Comment (EncryptedDev) [inline websites/Q/Quacky/presence.ts:32]: Maybe support the docs a bot more with dynamic DOM selectors

File: websites/S/statcord.com/presence.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
+const presence = new Presence({
+  clientId: "692810788196581376"
+}),
+
```
Comment (EncryptedDev) [inline websites/S/statcord.com/presence.ts:4]: Remove empty line

File: websites/S/statcord.com/presence.ts [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  if (document.location.hostname === "docs.statcord.com") 
+    presenceData.details = "Viewing Docs";
+  
+
+  if (presenceData.details === null) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else 
+    presence.setActivity(presenceData);
+
```
Comment (EncryptedDev) [inline websites/S/statcord.com/presence.ts:40]:
```suggestion
presence.setActivity(presenceData);
```

File: websites/S/statcord.com/presence.ts [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+      presenceData.details = `Viewing Page:`;
+      presenceData.state = "Bots List";
+    } else if (document.location.pathname.includes("/add")) {
+      presenceData.details = `Viewing Page:`;
+      presenceData.state = `Add Bot`;
+    }
+  }
+
+  if (document.location.hostname === "docs.statcord.com") 
+    presenceData.details = "Viewing Docs";
```
Comment (EncryptedDev) [inline websites/S/statcord.com/presence.ts:32]: Support docs better

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Low quality presence, should use query selectors to get dynamic elements from the page

File: websites/S/statcord.com/presence.ts (hunk 6 above)
Comment (Dom-in) [reply websites/S/statcord.com/presence.ts:32]: I did try with checking css and even geting location of mouse but still cant do that

Comment (Dom-in) [reply websites/S/statcord.com/presence.ts:30]: + i tryed using there `?id=quick-start` the think called id but did not work how i wanted did not check for it

Comment (Bas950) [reply websites/S/statcord.com/presence.ts:30]: document.location.hash

File: websites/Q/Quacky/presence.ts [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  if (document.location.hostname === "quacky.xyz") {
+    presenceData.details = "Viewing Page:";
+    presenceData.state = "Home";
+    if (document.location.pathname.includes("/vote")) {
+      presenceData.details = "Viewing Page:";
+      presenceData.state = "Vote";
+    } else if (document.location.pathname.includes("/login/")) {
+      presenceData.details = `Viewing Page:`;
```
Comment (EncryptedDev) [inline websites/Q/Quacky/presence.ts:18]:
```suggestion
      presenceData.details = "Viewing Page:";
```

File: websites/Q/Quacky/presence.ts [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+      presenceData.state = `Login`;
```
Comment (EncryptedDev) [inline websites/Q/Quacky/presence.ts:19]:
```suggestion
      presenceData.state = "Login";
```

File: websites/Q/Quacky/presence.ts [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+    } else if (document.location.pathname.includes("/profile")) {
+      presenceData.details = `Viewing Page:`;
```
Comment (EncryptedDev) [inline websites/Q/Quacky/presence.ts:21]:
```suggestion
      presenceData.details = "Viewing Page:";
```

File: websites/Q/Quacky/presence.ts [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+      presenceData.state = "Bots List";
+    } else if (document.location.pathname.includes("/add")) {
+      presenceData.details = `Viewing Page:`;
```
Comment (EncryptedDev) [inline websites/Q/Quacky/presence.ts:24]:
```suggestion
      presenceData.details = "Viewing Page:";
```

File: websites/Q/Quacky/presence.ts [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+      presenceData.state = `Add Bot`;
```
Comment (EncryptedDev) [inline websites/Q/Quacky/presence.ts:25]:
```suggestion
      presenceData.state = "Add bot";
```

File: websites/Q/Quacky/presence.ts [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  }
+
+  if (document.location.hostname === "docs.quacky.xyz") 
+    presenceData.details = "Viewing Docs";
+
```
Comment (EncryptedDev) [inline websites/Q/Quacky/presence.ts:31]:
```suggestion
```

File: websites/S/statcord.com/presence.ts [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+  };
+
+  if (document.location.hostname === "statcord.com") {
+    presenceData.details = "Viewing Page:";
+    presenceData.state = "Viewing At Home Page";
+    if (document.location.pathname.includes("/bot/")) {
+      presenceData.details = "Viewing:";
+      presenceData.state = `${document.querySelector("#overview > div:nth-child(1) > div > div > div > div.col.mr-2 > div.h5.mb-0.font-weight-bold").textContent} Statistics`;
+    } else if (document.location.pathname.includes("/login/")) {
+      presenceData.details = `Viewing Page:`;
```
Comment (EncryptedDev) [inline websites/S/statcord.com/presence.ts:18]:
```suggestion
      presenceData.details = "Viewing Page:";
```

File: websites/S/statcord.com/presence.ts [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+      presenceData.state = `Login`;
```
Comment (EncryptedDev) [inline websites/S/statcord.com/presence.ts:19]:
```suggestion
      presenceData.state = "Login";
```

File: websites/S/statcord.com/presence.ts [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+    } else if (document.location.pathname.includes("/profile")) {
+      presenceData.details = `Viewing Page:`;
```
Comment (EncryptedDev) [inline websites/S/statcord.com/presence.ts:21]:
```suggestion
      presenceData.details = "Viewing Page:";
```

File: websites/S/statcord.com/presence.ts [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+      presenceData.state = "Bots List";
+    } else if (document.location.pathname.includes("/add")) {
+      presenceData.details = `Viewing Page:`;
```
Comment (EncryptedDev) [inline websites/S/statcord.com/presence.ts:24]:
```suggestion
      presenceData.details = "Viewing Page:";
```

File: websites/S/statcord.com/presence.ts [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+      presenceData.state = `Add Bot`;
```
Comment (EncryptedDev) [inline websites/S/statcord.com/presence.ts:25]:
```suggestion
      presenceData.state = "Add bot";
```

File: websites/S/statcord.com/presence.ts [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+  }
+
+  if (document.location.hostname === "docs.statcord.com") 
+    presenceData.details = "Viewing Docs";
+  
+  if (presenceData.details === null) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else 
+    presence.setActivity(presenceData);
```
Comment (EncryptedDev) [inline websites/S/statcord.com/presence.ts:36]:
```suggestion
    presence.setActivity(presenceData);
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
you still use static details, try to use dynamic details by using query selector :/

**Pull request closed without merging**